src/Sylius/Component/Core/OrderProcessing/ShipmentFactoryInterface.php

```diff
@@ -21,9 +21,9 @@
 interface ShipmentFactoryInterface
 {
     /**
-     * Create shipment for order.
+     * Create shipments for order.
```

Comment:
Yes, in next iteration it will be more then one, we may revert this for a moment.